next/.site/layouts/partials/docs/inject/body.html

```diff
@@ -1,12 +1,25 @@
-<script
-  type="module"
-  src="https://cdn.jsdelivr.net/npm/@pwabuilder/pwaupdate"
-></script>
-
+<script type="module" src="https://cdn.jsdelivr.net/npm/@pwabuilder/pwaupdate"></script>
 <pwa-update swpath="{{ "/sw.js" | relURL }}"></pwa-update>
 
 {{- if default true .Site.Params.BookSearch }}
-{{- $searchJSFile := printf "%s.search.js" .Language.Lang }}
-{{- $searchJS := resources.Get "search.js" | resources.ExecuteAsTemplate $searchJSFile . | resources.Minify | resources.Fingerprint }}
-<script defer src="{{ $searchJS.RelPermalink }}" integrity="{{ $searchJS.Data.Integrity }}"></script>
-{{ end -}}
\ No newline at end of file
+  {{- $searchJSFile := printf "%s.search.js" .Language.Lang }}
+  {{- $searchJS := resources.Get "search.js" | resources.ExecuteAsTemplate $searchJSFile . | resources.Minify | resources.Fingerprint }}
+  <script defer src="{{ $searchJS.RelPermalink }}" integrity="{{ $searchJS.Data.Integrity }}"></script>
+{{ end -}}
+
+{{- if default false ($.Param "math-mode") -}}
+<script src="https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/katex.min.js" integrity="sha384-y23I5Q6l+B6vatafAwxRu/0oK/79VlbSz7Q9aiSZUvyWYIYsd+qj+o24G5ZU2zJz" crossorigin="anonymous"></script>
+<script src="https://cdn.jsdelivr.net/npm/katex@0.11.1/dist/contrib/auto-render.min.js" integrity="sha384-kWPLUVMOks5AQFrykwIup5lo0m3iMkkHrD0uJ4H5cjeGihAutqP0yW0J6dpFiVkI" crossorigin="anonymous"></script>
+<script>
+renderMathInElement(document.body, {
+    ignoredTags: ["script", "noscript", "style", "textarea"],
+    throwOnError: false,
+    delimiters: [
+        {left: "$$", right: "$$", display: true},
+        {left: "$", right: "$", display: false},
+        {left: "\\(", right: "\\)", display: false},
+        {left: "\\[", right: "\\]", display: true}
```

Comment:
We don't need those, I would just keep the dollar ones.

Reply:
we might in the future

Reply:
I prefer limiting things, so that you don't accidentally render things as math. With only dollar signs the rules are simple.